actions/views/notice.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Should be 2015-present?

Reply:
I thought it was supposed to be the year it was created

Reply:
Thanks for the reminder. I was confused about the latest agreement.